README.md

```diff
@@ -1,58 +1,48 @@
  - [Introduction.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)
  - [Introduction.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)
  - [Introduction_EN.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)
  - [Introduction_EN.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)
  - [Functional_list.xlsx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
 
-##詳細(details)
-以下のディレクトリを参照してください。
-(Please refer to the following directories.)
+## Details
+Please refer to the following directories.
 
  - [documents](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents)
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
 
-##内容物(contents)
+## Contents
 
-###ディレクトリ(directory)
+### Directory
 
 #### [/license/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/license)
-ライセンス(license)
+License
 
 #### [/root/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/root)
-プログラム
+Program
 
-##テンプレート・ベース(Templates base)
-テンプレート・ベースに同梱されるサンプルはOpen棟梁の評価に利用できます。
-また、テンプレート・ベースをチュートリアルの内容に従いカスタマイズすることで
-当該Visual Studioバージョンの案件向けプロジェクト・テンプレートを作成できます。 
-
-(Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
+## Templates base
+Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
 Further, by using templates base customized according to the contents of the tutorial, 
-you can create a project template for the project for the appropriate Visual Studio version.)
+you can create a project template for the project for the appropriate Visual Studio version.
 
-以下のリポジトリを参照してください。詳しくは、各フォルダのReadmeを参照下さい。
-(Please refer to the following repository. For more information, please refer to the Readme for each folder.)
+Please refer to the following repository. For more information, please refer to the Readme for each folder.
```

Comment:
@daisukenishino 

Please revise as follows:

For more information, please refer to the Readme files in the following repository.